src/Database/Adapter/Mongo.php

```
@@ -0,0 +1,2216 @@
+    }
+
+    protected function processException(Exception $e): \Exception
+    {
+        if ($e->getCode() === 50) {
+            return new Timeout('Query timed out', $e->getCode(), $e);
+        }
+
+        return $e;
+    }
+
+    protected function quote(string $string): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$string'. (Unused Code Rules)

(UnusedFormalParameter)